app/Concerns/HasPassword.php

```diff
@@ -92,7 +92,6 @@ public function removePassword(): bool
         return $this->fill([
             'password' => null,
             'password_set_at' => null,
-            'password_expires_at' => null,
```

Comment:
Why has this been removed? Would it be better to keep data consistently null across the board?